### PR TITLE
Homepg fixes

### PR DESCRIPTION
**Pull request by cjg89**
Last homepage adjustments before tagging v2.1.5.

### CODE REVIEW

File: dev/custom-pages/homepage.html [hunk 1]
```diff
@@ -1,194 +1,201 @@
...
-            <li><a href="//www.ucf.edu/academics/computer-science/">Computer Science</a></li>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8">
+        <h2>Get the degree you want.</h2>
+        <p class="description">UCF has <a href="https://www.ucf.edu/degree-search">more than 210 degree programs</a> in <a href="https://www.ucf.edu/academics/#colleges">13 colleges</a>, plus a talented team of faculty and staff to help find what’s right for you. Whatever your passion, we’ll put you on the path to success.</p>
+        <div class="degree-search-box">
+          <form action="//www.ucf.edu/degree-search/" method="GET">
+            <button type="submit" class="search-icon"><span class="fa fa-search"></span></button>
+            <input id="academics-degree-search" name="search-query" class="degree-search academics-search-icon" type="text" autocomplete="off" data-provide="typeahead" placeholder="Search for degree programs by name or area of interest" /></div>
```
Comment (rjucf) [inline dev/custom-pages/homepage.html]: This div should come after the < / form>

File: dev/custom-pages/homepage.html [hunk 2]
```diff
@@ -1,194 +1,201 @@
...
+      <div class="col-md-6 match-heights">
+        <h3 class="h4 heading-heavy"><a href="https://events.ucf.edu/upcoming/" class="highlight highlight-gold btn btn-gold" style="text-decoration: none">Up Next</a></h3>
+        [events url="https://events.ucf.edu/upcoming" start="0" limit="1" show_descriptions="true" use_short_month="true"]<p>No events found. Check the <a href="https://events.ucf.edu/upcoming/">full event calendar for future event details.</a></p>[/events]
+        <h3 class="h4 heading-heavy"><a href="https://events.ucf.edu" class="highlight highlight-gold btn btn-gold" style="text-decoration: none">Looking Ahead</a></h3>
+        [events url="https://events.ucf.edu/upcoming" start="1" limit="4" use_short_month="true"]<p>No events found. Check the <a href="https://events.ucf.edu/upcoming/">full event calendar for future event details.</a></p>[/events]
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12 hidden-xs hidden-sm">
+          <a href="https://events.ucf.edu" style="padding: .4em 0; color: #ffc904; background: transparent;">View All Events</a>
```
Comment (rjucf) [inline dev/custom-pages/homepage.html]: inline styles

Comment (cjg89) [inline dev/custom-pages/homepage.html]: Fixed here:
https://github.com/UCF/Main-Site-Theme/pull/19/commits/1b12e3faf983ebe4400daccac6ce27b23856f248

File: dev/custom-pages/homepage.html (hunk 1 above)
Comment (cjg89) [inline dev/custom-pages/homepage.html]: Good catch, thanks.  Fixed:
https://github.com/UCF/Main-Site-Theme/pull/19/commits/0ff7f9ea272a827b4253bb04f264b3984a34a14c

**Pull request merged**